test/Driver/linker.swift

```diff
@@ -265,6 +268,11 @@
 // DEBUG: linker
 // DEBUG: -o linker.dSYM
 
+// LINK-SWIFTMODULES: bin/swift
+// LINK-SWIFTMODULES-NEXT: bin/ld{{"? }}
+// LINK-SWIFTMODULES: -add_ast_path {{.*}}/a.swiftmodule
```

Comment:
The rest of this test doesn't do this yet, but FileCheck now has a `-SAME` suffix that guarantees that the check lines are on the same line as the previous match. Please use that here!